marklogic-data-hub-central/ui/src/components/record-view/record-view.test.tsx

```diff
@@ -3,9 +3,80 @@ import {render, fireEvent, waitForElement} from "@testing-library/react";
 import {entitySearch} from "../../assets/mock-data/explore/entity-search";
 import {BrowserRouter as Router} from "react-router-dom";
 import RecordCardView from "./record-view";
+import axiosMock from "axios";
+import {MemoryRouter} from "react-router-dom";
+import {SearchContext} from "../../util/search-context";
+
+jest.mock("axios");
 
 describe("Raw data card view component", () => {
 
+  const defaultSearchOptions = {
+    query: "",
+    entityTypeIds: [],
+    nextEntityType: "",
+    start: 1,
+    pageNumber: 1,
+    pageLength: 20,
+    pageSize: 20,
+    selectedFacets: {},
+    maxRowsPerPage: 100,
+    selectedQuery: "select a query",
+    zeroState: false,
+    manageQueryModal: false,
+    selectedTableProperties: [],
+    view: null,
+    sortOrder: [],
+    database: "final",
+  };
+
+  const response =
```

Comment:
This can also be moved to the same "non-entity-document-response.tsx" file that we can create to store the previous response , as mentioned in my earlier comment